script/test-in-bamboo.sh

```diff
@@ -68,3 +68,6 @@ cd test \
 	      EDL_USER="${EDL_USER}" \
 	      EDL_PASSWORD="${EDL_PASSWORD}" \
     && ./run_notebooks.sh
+
+# Copy the notebook artefacts up to S3:
+aws s3 cp output "s3://${REGRESSION_TEST_OUTPUT_BUCKET}" --recursive
```

Comment:
I think this should be the correct directory, as the previous line starts with a command to `cd test` (L64).

Reply:
The other output directory is getting removed and since this isn't a subshell command. Sounds like I believe you.